api/v01/entities/vrp_input.rb

```diff
@@ -479,7 +480,6 @@ module VrpVehicles
     optional(:force_start, type: Boolean, documentation: { hidden: true }, desc: '[ DEPRECATED ]')
     optional(:shift_preference, type: String, values: ['force_start', 'force_end', 'minimize_span'], desc: 'Force the vehicle to start as soon as the vehicle timewindow is open,
       as late as possible or let vehicle start at any time. Not available with periodic heuristic, it will always leave as soon as possible.')
-    optional(:trips, type: Integer, default: 1, desc: 'The number of times a vehicle is allowed to return to the depot within its route. Not available with periodic heuristic.')
```

Comment:
We can remove it directly since it wasn't even in the released docs (that's the case right?) so it didn't exist from the point of view of the end user. If it was released, I still want to say, we can just get rid of it if there is no one using it.